### PR TITLE
Implement explicit Keplr wallet connect/disconnect

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/873

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles 
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-components/src/components/header/UserMenuItem.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                sx={{ height: 18, width: 20 }}
+                color={theme.palette.secondary.main}
+              />
+            ),
+          },
+        ],
+        extras: (
+          <Box sx={{ mx: -3.5 }}>
+            <Separator />
+            {/* TODO replace with Label component from David's PR #906 */}
```
Comment (blushi) [inline web-components/src/components/header/UserMenuItem.tsx:71]: cc/ @haveanicedavid depends whether your PR or mine gets merged first

File: web-registry/src/lib/wallet.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, createContext } from 'react';
```
Comment (blushi) [inline web-registry/src/lib/wallet.tsx:1]: this is the most important diff to review

**Review by mhagel — APPROVED** [review]
One thing I noticed. If user is in a browser that does not support Keplr (Safari), the Connect Wallet button is still visible. When you click it, it goes away. I think this is ok for now, but eventually we may want to add a popup that says "please use a chrome-based browser and download the keplr extension" or something.

**Comment by blushi** [comment]
> One thing I noticed. If user is in a browser that does not support Keplr (Safari), the Connect Wallet button is still visible. When you click it, it goes away. I think this is ok for now, but eventually we may want to add a popup that says "please use a chrome-based browser and download the keplr extension" or something.

Good catch, fixed in my latest commits.

**Comment by wgwz** [comment]
I wanted to point out something that I found a little confusing in the UI. (I was testing at this deploy preview: https://deploy-preview-916--regen-registry.netlify.app/

So we've got this as the first step, which went more or less as I expected:

Screenshot_2022-05-02_15-42-35

Then this what I see on the next screen after unlocking (which was unexpected, I was expecting to see an approval screen here):

Screenshot_2022-05-02_15-43-06

I didn't see the approval screen until I clicked the "connect" button again (and also I'm seeing that error message in the background):

Screenshot_2022-05-02_15-43-47

After that it looks good:

Screenshot_2022-05-02_15-44-16

**Review by haveanicedavid — APPROVED** [review]

**Comment by blushi** [comment]
@wgwz Somehow I can't reproduce what you describe, I suspect this might have something to do with your local setup, in particular the extension related error you were getting during our 1-1 yesterday. There might be something wrong with your chrome/keplr setup.

**Comment by blushi** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**